docker/Dockerfile

```diff
@@ -42,17 +45,18 @@ RUN npm ci
 
 # Base arguments: build-time
 ARG NPM_CONFIG_LOGLEVEL=warn
-ARG PORT=3000
+ARG PORT=8787
```

Comment:
why did you change this port?
I think it can potential affect our staging/production deploy.

Reply:
Done! ✅